modules/op-resolver-community-tasks-ref.adoc

```diff
@@ -0,0 +1,411 @@
+        - name: INSECUREREGISTRY
+          value: 'false'
+        - name: CACERTFILE
+          value: service-ca.crt
+      taskRef:
+        kind: Task
+        name: jib-maven
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+  #...
+----
+
+.Supported parameters for the `jib-maven` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`IMAGE` |Name of the image to build. |
+|`MAVEN_IMAGE` |Maven base image. |`registry.redhat.io/ubi9/openjdk-17@sha256:78613bdf887530100efb6ddf92d2a17f6176542740ed83e509cdc19ee7c072d6`
+|`DIRECTORY` |The{nbsp}directory{nbsp}containing{nbsp}the{nbsp}app,{nbsp}relative{nbsp}to the source repository root. |`.`
+|`CACHE` |The name of the volume for caching Maven artifacts and base image layers. |`empty-dir-volume`
+|`INSECUREREGISTRY` |Allow an insecure registry. |`false`
+|`CACERTFILE` |Certificate authority (CA) bundle file name for an insecure registry service. |`service-ca.crt`
+|===
+
+.Supported workspaces for the `jib-maven` community task
+[cols="1,3",options="header"]
+|===
+| Workspace | Description
+| `source` | The workspace that contains the Maven project.
+|`sslcertdir` | The optional workspace that contains SSL certificates.
+|===
+
+.Results that the `jib-maven` task returns
+[cols="1,1,2",options="header"]
+|===
+| Result | Type | Description
+|`IMAGE_DIGEST` |`string` |Digest of the image that was built.
+|===
+
+.Changes from the `jib-maven` community cluster task
+
+* The default values for the `IMAGE` and `MAVEN_IMAGE` parameters were changed.
+
+[discrete]
+[id="op-ctaskref-kubeconfig-creator_{context}"]
+== kubeconfig-creator
+The `kubeconfig-creator` community task creates a `kubeconfig` file that can be used by other tasks in the pipeline to access different clusters.
+
+.Example usage of the `kubeconfig-creator` community task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: kubeconfig-creator
+spec:
+  tasks:
+    - name: kubeconfig-creator
+      params:
+        - name: name
+          value: example_cluster
+        - name: url
+          value: example_cluster_URL
+        - name: username
+          value: example_username
+        - name: password
+          value: example_password
+        - name: cadata
+          value: ''
+        - name: clientKeyData
+          value: ''
+        - name: clientCertificateData
+          value: ''
+        - name: namespace
+          value: ''
+        - name: token
+          value: ''
+        - name: insecure
+          value: 'false'
+      taskRef:
+        kind: Task
+        name: kubeconfig-creator
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+  #...
+----
+
+.Supported parameters for the `kubeconfig-creator` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`name` |The name of the cluster to access. |
```

Comment:
Remove "The" here and in the other parameters, I think we don't normally use an article at the start of a parameter description in the table (compare to how the non-community tasks are done)